src/main/resources/com/google/api/codegen/ruby/version_index.snip

```diff
@@ -1,12 +1,59 @@
+@extends "readme.snip"
 @extends "ruby/common.snip"
+@extends "ruby/method_sample.snip"
 
 @snippet generate(index)
-  {@toComments(index.fileHeader.copyrightLines)}
-  #
-  {@toComments(index.fileHeader.licenseLines)}
+  {@license(index.fileHeader)}
+
+  @if index.modules
+    {@modules(index.modules.iterator)}
+  @end
 
   @join requireView : {@index.requireViews}
       require "{@requireView.fileName}"
   @end
 
 @end
+
+@private modules(iterator)
```

Comment:
It makes the indentation easier. These are nested modules rather than sequential.